reloj-checador/app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,65 +1,58 @@
-// body {
-//   background-color: #fff;
-//   color: #333;
-//   margin: 33px; }
-
-// body, p, ol, ul, td {
-//   font-family: verdana, arial, helvetica, sans-serif;
-//   font-size: 13px;
-//   line-height: 18px; }
-
-// pre {
-//   background-color: #eee;
-//   padding: 10px;
-//   font-size: 11px; }
-
-// a {
-//   color: #000; }
-
-// a:visited {
-//   color: #666; }
-
-// a:hover {
-//   color: #fff;
-//   background-color: #000; }
-
-// th {
-//   padding-bottom: 5px; }
-
-// td {
-//   padding: 0 5px 7px; }
-
-// div.field,
-// div.actions {
-//   margin-bottom: 10px; }
-
-// #notice {
-//   color: green; }
-
-// .field_with_errors {
-//   padding: 2px;
-//   background-color: red;
-//   display: table; }
-
-// #error_explanation {
-//   width: 450px;
-//   border: 2px solid red;
-//   padding: 7px 7px 0;
-//   margin-bottom: 20px;
-//   background-color: #f0f0f0; }
-
-// #error_explanation h2 {
-//   text-align: left;
-//   font-weight: bold;
-//   padding: 5px 5px 5px 15px;
-//   font-size: 12px;
-//   margin: -7px -7px 0;
-//   background-color: #c00;
-//   color: #fff; }
-
-// #error_explanation ul li {
-//   font-size: 12px;
-//   list-style: square; }
-
-// label {
-//   display: block; }
+*{
```

Comment:
Do we really want to apply these styles to every single element in the DOM?
Why is this necessary?

Reply:
Same as above